src/main/java/com/ericsson/ei/notifications/EmailSender.java

```diff
@@ -48,17 +48,31 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(EmailSender.class);
 
     @Getter
-    @Value("${email.sender}")
-    private String sender;
+    @Value("${email.sender:noreply@ericsson.com}")
+    private static final String SENDER = "noreply@ericsson.com";
 
     @Getter
-    @Value("${email.subject}")
-    private String subject;
+    @Value("${email.subject:Email Subscription Notification}")
+    private static final String SUBJECT = "Email Subscription Notification";
 
     @Autowired
     private JavaMailSender emailSender;
 
     /**
+     * @return the sender
+     */
+    public String getSender() {
```

Comment:
EI uses lombok @Getter so you don't beed to explicitly define these types of methods. You can read about it here: https://www.baeldung.com/lombok-ide
You may be getting an error currently and/or be able unable to get these varibles unless installing lombok.
Once you do you won't need these methods anymore.

Reply:
okay